pkg/utils/utils_test.go

```diff
@@ -57,4 +60,32 @@ var _ = Describe("Utils", func() {
 			})
 		})
 	})
+
+	Describe("#GenerateEtcdOwnerReference", func() {
+		It("should return expected ownerRefs ", func() {
+			etcd := &druidv1alpha1.Etcd{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "etcd-name",
+					Namespace: "etcd-namespace",
+					UID:       "etcd-uid",
+				},
+			}
+			expected := []metav1.OwnerReference{
+				{
+					APIVersion:         druidv1alpha1.GroupVersion.String(),
+					Kind:               etcd.Kind,
+					Name:               etcd.Name,
+					UID:                etcd.UID,
+					Controller:         pointer.Bool(true),
+					BlockOwnerDeletion: pointer.Bool(true),
+				},
+			}
+			ownerRefs := GenerateEtcdOwnerReference(nil)
```

Comment:
```suggestion
			ownerRefs := GenerateEtcdOwnerReference(etcd)
```
To fix failing test case